Controller/articles.controller.js

```diff
@@ -25,3 +27,22 @@ exports.getAllArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVoteById = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const updateBody = req.body;
+
+  const promises = [updateArticleVoteById(articleId, updateBody)];
+
+  if (articleId) {
+    promises.push(checkExist("articles", "article_id", articleId));
```

Comment:
We don't actually need to be checking the article exists here. You can just throw a custom error from your model 👍